san/sanbase_graphql.py

```diff
@@ -228,20 +245,17 @@ def mining_pools_distribution(idx, slug, **kwargs):
 
 
 def historical_balance(idx, slug, **kwargs):
-    kwargs = _transform_query_args(**kwargs)
+    kwargs = _transform_query_args("historical_balance", **kwargs)
 
-    query_str = """
+    query_str = ("""
     query_{idx}: historicalBalance (
         address: \"{address}\",
         slug: \"{slug}\",
         from: \"{from_date}\",
         to: \"{to_date}\",
         interval: \"{interval}\"
     ){{
-        balance,
-        datetime
-    }}
-    """.format(
+    """ + ' '.join(kwargs['return_fields']) + '}}').format(
```

Comment:
How does `join` know that it needs to convert the tuple  ` ('fromAddress', ['address', 'isExchange'])` to `fromAddress{address isExchange}` :?

Reply:
Hmm, good question. https://github.com/santiment/sanpy/pull/78/files/dfd80d394fc15a935a3fdc79a3f32e551c7a49e6#diff-0f5457262476aa06c3885256259a709dR335
I will do something similar here, thanks for the heads-up

Reply:
It works (I have tested it locally) - I just do not know why

Reply:
Now it should know